server/jetstream_errors_generated.go

```diff
@@ -446,6 +449,7 @@ var (
 		JSConsumerMaxRequestBatchNegativeErr:       {Code: 400, ErrCode: 10114, Description: "consumer max request batch needs to be > 0"},
 		JSConsumerMaxRequestExpiresToSmall:         {Code: 400, ErrCode: 10115, Description: "consumer max request expires needs to be >= 1ms"},
 		JSConsumerMaxWaitingNegativeErr:            {Code: 400, ErrCode: 10087, Description: "consumer max waiting needs to be positive"},
+		JSConsumerMetadataLengthErrF:               {Code: 400, ErrCode: 10134, Description: "Consumer metadata exceeds maximum size of {limit}"},
```

Comment:
```suggestion
		JSConsumerMetadataLengthErrF:               {Code: 400, ErrCode: 10134, Description: "consumer metadata exceeds maximum size of {limit}"},
```

Reply:
casing issue here, the error below has the same issue though...

Reply:
yes, fixed.